pkg/cmd/workspace/delete.go

```diff
@@ -18,11 +19,31 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
```

Comment:
I'd love if we were consistent when declaring vars.

In the rest of the files you used:
```go
var X
var Y
```
and here you used:
```go
var (
 X,
 Y
)
```